cypress/app/app.js

```diff
@@ -103,6 +103,36 @@ app.get('/about', (req, res) => res.send(`
 `));
 
 
+app.get('/elements', (req, res) => res.send(`
+  <html>
+    <head>
+      <title>Elements Swap</title>
+      <meta property="og:url" content="http://127.0.0.1:8888/about">
+      <meta property="og:title" content="Elements | Swap">
+      <meta charset="UTF-8">
+      <script src="/${frontendJS}" type="application/javascript"></script>
+      <link rel="stylesheet" href="/${mainCSS}">
+    </head>
+    <body>
+    ${menu}
+    ${layout(`
+      <a href="/replace-element" data-swap=".content <> .replacement">Replacement Swap</a>
```

Comment:
@shanebo I realized we never discussed the order of the selectors with the replace operators. This is the approach that made sense to me:

`.content <> .replacement` which means "replace `.content` in the old document with `.replacement` from the new document"

Let me know if you were thinking of the reverse order, which would be:
`.replacement <> .content`, which I think would mean "`.replacement` from the new document replaces `.content` in the old document"